addons/point_of_sale/models/pos_order.py

```diff
@@ -290,6 +290,7 @@ def _prepare_invoice_lines(self):
                 'name': line_values['name'],
                 'tax_ids': [(6, 0, line_values['taxes'].ids)],
                 'product_uom_id': line_values['uom'].id,
+                'analytic_distribution': line_values['analytic_distribution'],
```

Comment:
if pos_sale is not installed, this line can crash. Maybe extract a method that returns this 'invoice line vals'.